pkg/chunk/cassandra/storage_client.go

```diff
@@ -97,11 +102,23 @@ func (cfg *Config) setClusterConfig(cluster *gocql.ClusterConfig) {
 		}
 	}
 	if cfg.Auth {
+		if cfg.Password != "" && cfg.PasswordFile != "" {
+			return errors.Errorf("The -cassandra.password and -cassandra.password-file flags are mutually exclusive.")
+		}
```

Comment:
I would remove this in favour of `Validate()` which is the pattern we're now using to validate the config.